nextflow_schema.json

```diff
@@ -647,12 +649,14 @@
                     "type": "boolean",
                     "description": "Display help text.",
                     "fa_icon": "fas fa-question-circle",
+                    "default": false,
```

Comment:
You can remove those `"default": false` as they are removed by default when running `nf-core schema build`

Reply:
all the default: false were removed.